app/blueprints/main/postgres_database_manager.py

```diff
@@ -92,4 +92,23 @@ def upsert_user_by_github_user_id(self, username: str, github_user_id: int, acce
             SET access_token = %s, access_token_expires_in = %s, refresh_token = %s, refresh_token_expires_in = %s
         """
         params = (username, github_user_id, access_token, access_token_expires_in, refresh_token, refresh_token_expires_in, access_token, access_token_expires_in, refresh_token, refresh_token_expires_in)
-        self._execute_query(query, params)
\ No newline at end of file
+        self._execute_query(query, params)
+    
+    def upsert_lingo_by_github_user_id(self, github_user_id: int, name: str, style: str, has_steps: bool, has_impact: bool, has_location: bool, has_expected: bool, has_culprit: bool) -> None:
```

Comment:
Hey @Ricram2,
Everything looks great! I was just wondering if this should be 'insert_', instead of 'upsert_'?